docs/examples/catch.md

```diff
@@ -0,0 +1,200 @@
+# Catch
+
+```blocks
+enum SpriteKind {
+    Player,
+    Projectile,
+    Food,
+    Enemy,
+    Projectile2,
+    snake
+}
+let sprite_list: Sprite[] = []
+let limit = 0
+let basket: Sprite = null
+let mySprite4: Sprite = null
+let falling: Sprite = null
+let s4Dir = 0
+let otherSprite: Sprite = null
```

Comment:
Also want to drop the declarations for otherSprite and sprite here; they show up as being assigned to null in the ``on start``, as they're only used in the overlaps below as parameters

Reply:
Done.